requirements-docs.txt

```diff
@@ -1 +1,3 @@
 Sphinx==1.3.1
+sphinx-django-command
```

Comment:
nice, I didn't know this -- thanks for adding it and simplifying / improving the docs :)